.storybook/components/Docs/Guidelines/Theming.stories.mdx

```diff
@@ -10,7 +10,91 @@ import { Canvas, Meta } from '@storybook/blocks';
 
 ## How to apply a theme in another product
 
-In EDS, theming is implemented by overriding the values of the CSS variables representing tokens, which the EDS components use in their styles. This should update the style of the components to match the branding of a different product with minimum manual CSS styling overrides. (Some manual styling overrides will be necessary though because we don't have tokens for every little detail. In those cases, we could create a new token to make those overrides easier if it looks like something that could very well be useful for other products as well.)
+EDS comes with some tooling to allow easy transfer of theme data from Figma (or some style-dictionary compatible format) into code.
+
+* `eds-init-theme` - This command sets up the initial file(s) for theming your application
+* `eds-apply-theme` - This command parses the style dictionary files to generate the tokens (CSS Variables) used by EDS
+
+Each of these tools reads config to figure out where to read/write files. This can be defined in several ways, e.g., a top-level file `eds.config.js`, or as a key-value set in package.json. Example:
+
+`package.json`
+
+```json
+"eds": {
+  "json": "src/components/",
+  "css": "src/components/"
+},
+```
+
+`json` determines where the core theme file will be stored, and `css` determines where the resulting css token file will be stored.
+
+### eds-init-theme
```

Comment:
nit: can we name the files generated by `eds-init-theme` and specify which one to modify?

Reply:
👍🏾! added in file names in appropriate spots and better sequencing in the doc.s